clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -206,6 +209,8 @@ synchronized public void addTopic(boolean internal,
                 logDirs.add(brokerLogDirs.get(partition.leader().id()).get(0));
             }
         }
+        Uuid topicId = Uuid.randomUuid();
+        topicIds.put(name, topicId);
```

Comment:
Do we want the option to test with and without topic ids?

Reply:
Currently, MockAdminClient is mainly used in connect module and stream module, we could add another ticket to do this.

Reply:
Yes, I add a method with a `usesTopicId` parameter.